code/optimize/src/main/java/com/adobe/marketing/mobile/optimize/Optimize.java

```diff
@@ -90,7 +90,7 @@ public static void updatePropositions(
             @Nullable final Map<String, Object> data,
             @Nullable final AdobeCallback<Map<DecisionScope, OptimizeProposition>> callback) {
         final double defaultTimeoutSeconds =
-                OptimizeConstants.EDGE_CONTENT_COMPLETE_RESPONSE_TIMEOUT;
+                (double) Long.MAX_VALUE / OptimizeConstants.TIMEOUT_CONVERSION_FACTOR;
```

Comment:
Why are we dividing here and again multiplying in `retrieveOptimizeRequestTimeout`? Could we not just send `Long.MAX_VALUE` as is in event data?